configure.sh

```diff
@@ -21,10 +21,12 @@ function write_action_env_to_bazelrc() {
   write_to_bazelrc "build --action_env $1=\"$2\""
 }
 
-rm .bazelrc
+[ -f .bazelrc ] && rm .bazelrc
 
 # TODO: Verify the tensorflow version here...
-if python -c "import tensorflow" &> /dev/null; then
+TENSORFLOW_VERSION=$(python -c 'import tensorflow as tf; print(tf.__version__)')
+if [[ $TENSORFLOW_VERSION == 2* ]] 
+then
     echo 'using installed tensorflow'
 else
     pip install tf-nightly-2.0-preview
```

Comment:
Perhaps we don't need to check tf version, just always `pip install --upgrade  tf-nightly-2.0-preview`?

Reply:
Good point, that's valid while we're still in preview. Once we have releases I think having a check will be a good idea though so I'm okay with adding it now.

Reply:
You're right, I agree! How about creating requirements.txt? with `pip install -r requirements.txt`?

Reply:
I like that idea, its much more pythonic. @armando-fandango Could you modify the script so it always runs a pip install from `requirements.txt`, and then just specify the version in the requirements file.

The check for `.bazelrc` is still a good addition

Reply:
Yes, thanks for the review and suggestions. Let me move pip install to requirements.txt.